### PR TITLE
DMC 6 Cards

### PR DESCRIPTION
**Pull request by Simisays**
The only problem I'm having is with Stangg. I can't figure out how to to combine the exile triggers of the copies and the Stangg token. 

-Orca, Siege Demon
-Gerrard's Hourglass Pendant
-Tetsuo, Imperial Champion
-Torsten, Founder of Benalia
-Tobias, Doomed Conqueror
-The Lady of Otaria

### CODE REVIEW

**Comment by Hanmac** [comment]
for Stagg, `AtEOT$ Exile` sadly can't be used, try using Imprint. @Northmoc probably can help to make the right trigger

**Comment by Hanmac** [comment]
yeah, that's exactly why i said that AtEOT can't be used

File: forge-gui/res/cardsfolder/upcoming/tobias_conqueror_of_bad_luck.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Name:Tobias, Doomed Conqueror
+ManaCost:2 W U
+Types:Legendary Creature Human Soldier
+K:Flash
+PT:3/2
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ When CARDNAME dies, create a 2/2 black Zombie creature token for each nontoken creature you controlled that died this turn.
+SVar:TrigToken:DB$ Token | TokenAmount$ X | TokenScript$ b_2_2_zombie | TokenOwner$ You
+SVar:X:Count$ThisTurnEntered_Graveyard_from_Battlefield_Creature.nonToken+YouOwn
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/tobias_conqueror_of_bad_luck.txt:8]: YouCtrl

File: forge-gui/res/cardsfolder/upcoming/orca_siege_demon.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+ManaCost:5 B R
+Types:Legendary Creature Human Soldier
+K:Trample
+PT:5/5
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Creature.Other | TriggerZones$ Battlefield | Execute$ TrigPutCounter | TriggerDescription$ Whenever another creature dies, put a +1/+1 counter on CARDNAME.
+SVar:TrigPutCounter:DB$ PutCounter | Defined$ Self | CounterType$ P1P1 | CounterNum$ 1
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Card.Self | Execute$ TrigDamage | TriggerDescription$ When NICKNAME dies, it deals damage equal to its power divided as you choose among any number of targets. 
+SVar:TrigDamage:DB$ DealDamage | ValidTgts$ Creature,Player,Planeswalker | TgtPrompt$ Select any number of targets to distribute damage to | NumDmg$ X | TargetMin$ 1 | TargetMax$ X | DividedAsYouChoose$ X
+SVar:X:TriggeredCard$CardPower
+Oracle:Whenever another creature dies, put a +1/+1 counter on Orca, Siege Demon.\nWhen Orca dies, it deals damage equal to its power divided as you choose among any number of targets.
```
Comment (Agetian) [inline forge-gui/res/cardsfolder/upcoming/orca_siege_demon.txt:11]: DeckHas:Ability$Counters

File: forge-gui/res/cardsfolder/upcoming/torsten_founder_of_benalia.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+Name:Torsten, Founder of Benalia
+ManaCost:5 G W
+Types:Legendary Creature Human Soldier
+PT:7/7
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigDig | TriggerDescription$ When CARDNAME enters the battlefield, reveal the top seven cards of your library. Put any number of creature and/or land cards from among them into your hand and the rest on the bottom of your library in a random order.
+SVar:TrigDig:DB$ Dig | DigNum$ 7 | ChangeValid$ Creature,Land | DestinationZone$ Hand | RestRandomOrder$ True | AnyNumber$ True
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ When NICKNAME dies, create seven 1/1 white Soldier creature tokens.
+SVar:TrigToken:DB$ Token | TokenAmount$ 7 | TokenScript$ w_1_1_soldier | TokenOwner$ You
+Oracle:When Torsten, Founder of Benalia enters the battlefield, reveal the top seven cards of your library. Put any number of creature and/or land cards from among them into your hand and the rest on the bottom of your library in a random order.\nWhen Torsten dies, create seven 1/1 white Soldier creature tokens.
```
Comment (Agetian) [inline forge-gui/res/cardsfolder/upcoming/torsten_founder_of_benalia.txt:9]: DeckHas:Ability$Token

**Comment by Simisays** [comment]
Created a new PR for Stangg and removed that card from this PR so the other cards can be merged

File: forge-gui/res/cardsfolder/upcoming/gerrards_hourglass_endant.txt [hunk 4]
```diff
@@ -0,0 +1,7 @@
+Name:Gerrard's Hourglass Pendant
+ManaCost:1
+Types:Legendary Artifact
+K:Flash
+R:Event$ BeginTurn | ActiveZones$ Battlefield |  ExtraTurn$ True | Skip$ True | Description$ If a player would begin an extra turn, that player skips that turn instead.
+A:AB$ ChangeZone | Cost$ 4 T Exile<1/CARDNAME> | Origin$ Graveyard | Destination$ Battlefield | Defined$ ValidGraveyard | Tapped$ True | Artifact.YouOwn+ThisTurnEnteredFrom_Battlefield,Creature.YouOwn+ThisTurnEnteredFrom_Battlefield,Land.YouOwn+ThisTurnEnteredFrom_Battlefield,Enchantment.YouOwn+ThisTurnEnteredFrom_Battlefield | SpellDescription$ Return to the battlefield tapped all artifact, creature, enchantment, and land cards in your graveyard that were put there from the battlefield this turn.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/gerrards_hourglass_endant.txt:6]: The Defined$ got messed up

**Review by Agetian — APPROVED** [review]

**Pull request merged**